infra/charts/turing/values.yaml

```diff
@@ -108,6 +135,9 @@ dbMigrations:
     # -- Docker tag for golang-migrate Docker image https://hub.docker.com/r/migrate/migrate
     tag: v4.7.1
 
+sentry:
+  dsn: ""
```

Comment:
Maybe a comment to say what this is and regenerate the helm-docs?

Reply:
As it's said in PR description, docs will be addressed in a separate PR